homeassistant/components/fibaro/sensor.py

```diff
@@ -65,6 +71,8 @@ async def async_setup_entry(
     entities: list[SensorEntity] = []
     for device in hass.data[DOMAIN][entry.entry_id][FIBARO_DEVICES]["sensor"]:
         entities.append(FibaroSensor(device))
+        if "power" in device.interfaces:
```

Comment:
Instead of detecting power sensor here again, wouldn't it be better to add sensor platform in the loop below wher this detection is already made?
So instead of `for device_type in ("cover", "light", "switch"):` you would use `for device_type in ("cover", "light", "switch", "sensor"):`

Reply:
Didn't want to loop over sensors again but ok, I've change that.